packages/eslint-plugin-svelte/src/rules/require-store-callbacks-use-set-param.ts

```diff
@@ -1,5 +1,36 @@
+import { getScope } from '../utils/ast-utils.js';
+import type { RuleContext, SuggestionReportDescriptor } from '../types.js';
 import { createRule } from '../utils/index.js';
 import { extractStoreReferences } from './reference-helpers/svelte-store.js';
+import type { TSESTree } from '@typescript-eslint/types';
+import type { Variable } from '@typescript-eslint/scope-manager';
+import { getSourceCode } from '../utils/compat.js';
+
+function findVariableForName(
```

Comment:
this is a duplicate of #1163 so i'll move it into a util once that one lands and update this